### DISCUSSION TITLE
Provide a way to embed a GitHub project

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#17289] [Feature Requests]
We allow multiple types of GitHub embeds:
- Profiles
- Gists
- Issues
- Repos

But, we don't provide a way to embed [GitHub projects](https://docs.github.com/en/issues/trying-out-the-new-projects-experience) which are relatively new and in beta.

I think it would be cool if there was a way to embed projects.

**Additional Context**

An org admin asked if this was possible and it didn't appear to be when I tested, so this request is coming from the community.

**Comment by jeremyf** [comment]
To confirm, I went looking at a project's HTML metadata regarding it's twitter card and open graph, and there's nothing useful in that information.  Which would mean a customized liquid tag.

I think we could point the community to https://github.com/forem/forem/blob/main/app/liquid_tags/github_tag.rb for our implementation pattern.  This would be a bit of Ruby work and some decisions around what to show.

**Reply by michael-tharrington** [reply]
Good deal! That totally makes sense. Appreciate you looking into this!

**Reply by amywtlin** [reply]
@jeremyf How do you feel about this effort-wise? If it is small, we can tackle it as a small win. If not, I am moving this to `icebox`.

**Reply by jeremyf** [reply]
@amywtlin I think this is relatively small and contained in scope.  And there's prior art which should make a new feature easier.

**Reply by amywtlin** [reply]
~@jeremyf Great, I can create an issue for this. You mentioned that this will be customized and we need to decide what to show. Can you point me to a list of available data where I can see what we should show, given our current design?~

~@ludwiczakpawel Is [this](https://www.figma.com/file/fJuPU4FRJQw63virkPiC9f/Liquid-tags?node-id=502%3A53) the correct finalized design spec for the embed layout?~

I created [an issue](https://github.com/forem/forem/issues/17688) for this. Let's move discussions over there.
